src/js/utils/dom-utils.js

```diff
@@ -112,6 +111,11 @@ function getAttributesArray(element) {
   return result;
 }
 
+function addClassName(element, className) {
+  // FIXME-IE IE10+
```

Comment:
yeah — I'm adding `FIXME-IE` as breadcrumbs for an IE audit at some point in the future.